doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -46,7 +46,12 @@ If your authoring environment uses a built-in {CENTRAL} Maven repository, change
 +
 ** In some authoring environments, you might need to ensure that several users can deploy services on the same {KIE_SERVER} at the same time. By default, after deploying a service onto a {KIE_SERVER} using {CENTRAL}, the user needs to wait for some seconds before more services can be deployed. To remove this limitation, you can configure the environment to use the _controller strategy_. Do not make this change unless a specific need for it exists; if you decide to enable controller strategy, make this change on {CENTRAL} and on all {KIE_SERVERS} in the same environment.
 +
-To enable controller strategy on {CENTRAL}, set the  `KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED` environment variable to `false`.
+[NOTE]
+====
+Do not enable the controller strategy in an environment with a high-availability {CENTRAL}. In such environments the controller strategy does not function correctly. 
```

Comment:
Suggest: Do not enable the controller strategy in a high-availability environment in {CENTRAL}. If enabled, the controller strategy does not function correctly.